test/recur_iterator_test.js

```diff
@@ -226,6 +227,38 @@ suite('recur_iterator', function() {
       assert.deepEqual(dates, options.dates || []);
     });
   }
+
+  function testRRULEWithError(ruleString, options, expectedError) {
```

Comment:
It doesn't look like `expectedError` gets a value, I'm assuming if these tests pass then `assert.throws(...)` only takes two parameters?